projects/packages/my-jetpack/_inc/components/plans-section/index.jsx

```diff
@@ -50,7 +50,7 @@ function PlanSectionHeader( { purchases } ) {
 					: __( 'Your plans', 'jetpack-my-jetpack' ) }
 			</H3>
 			{ purchases.length === 0 && (
-				<p>{ __( 'The extra power you added to your Jetpack.', 'jetpack-my-jetpack' ) }</p>
+				<Text variant="body">{ __( 'Want to power up your Jetpack?', 'jetpack-my-jetpack' ) }</Text>
```

Comment:
`body` is already the default `variant`, so you don't need to set `variant="body"`

Reply:
🤦 going to clean them in a follow-up. Thanks.